python-sdk-sample/README.md

```diff
@@ -1,7 +1,35 @@
 # Zowe Python SDK Sample
 
+## Requirements
+
+The sample SDK has the following requirements:
+```
+black
+isort
+zowe.core-for-zowe-sdk~=1.0.0.dev
+pytest==7.1.2
+```
+You can install the requirements in a virtual environment with following commands:
+
 ```shell
 python3 -m venv venv
 source venv/bin/activate
 pip install -r requirements.txt
 ```
+
+You can clone the repository using `git`:
+
+```
+git clone https://github.com/zowe/zowe-client-samples.git
+```
+
+## Quickstart
+
+After you install the package, import the class SampleSDK from modules
+Below is a simple script of how to run the sample SDK:
+
+```python
+t = SampleSdk()
```

Comment:
Please include the import for sample SDK. The example should be a complete Python script.